tests/scripts/run_demos.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python3
+"""Run the Mbed TLS demo scripts.
+"""
+import glob
+import subprocess
+import sys
+
+def run_demo(demo):
+    """Run the specified demo script. Return True if it succeeds."""
+    returncode = subprocess.call([demo])
```

Comment:
I am not sure I understand why `demo` is being passed to `subprocess.call` as an optional parameter?

Reply:
The first argument to `call` is the program's command line. It's a list where the first element is the program name, which is used as the path to the executable (with `$PATH` lookup) unless there's a separate argument for that.

Reply:
Thanks